src/daemon/led_mouse_bragi.c

```diff
@@ -24,6 +24,20 @@ int updatergb_mouse_bragi(usbdevice* kb, int force){
 
     uchar pkt[BRAGI_JUMBO_SIZE] = {0};
 
+    if (kb->vendor == V_CORSAIR && kb->product == P_KATAR_PRO_XT) {
+        uchar response[BRAGI_JUMBO_SIZE] = {0};
+
+        // Its ATM unknown how to set the color for the wheel for this mouse, but we can set its brightness so this is used to allow turning the LED on and off
+        int brightness = newlight->g[LED_MOUSE + 3] * 1000 / 255;
+        pkt[0] = BRAGI_MAGIC;
+        pkt[1] = BRAGI_SET;
+        pkt[2] = BRAGI_BRIGHTNESS;
+        pkt[4] = brightness & 255;
+        pkt[5] = brightness >> 8;
+        usbrecv(kb, pkt, sizeof(pkt), response);
```

Comment:
Please do not use usbrecv directly. Use `bragi_set_property()` instead.

Reply:
Ah, I see now. It takes a `const uchar` but brightness is a ushort. Feel free to modify that function to accept a ushort.

I think there's another stray usbrecv in there due to ushort, but I didn't think it was worth doing it for only a single call, but there's another one now, so it's fine.

Reply:
changed the  bragi_set_property() code to ushort and used it. The get code maybe should be changed too, also the DPI0-2 colors (nothing is done with these) are 3x8bit IIRC so iam not sure ushort will always be large enough